int8 motor_id
int8 temperature
float64 position
float64 speed
float64 torque_current
float64 torque